### PR TITLE
Auto-clean no-hijack BGP updates (Closes #33)

### LINKED ISSUE #33: Discard non-hijack BGP updates older than a certain age
**Is your feature request related to a problem? Please describe.**
Benign BGP updates that are older than a certain age are not currently deleted.

**Describe the solution you'd like**
ENV variable (e.g., DEL_NON_HIJ_UPDATES_AFTER) to specify maximum age for benign BGP updates, and functionality (backend) that deletes
the old BGP updates (only those not related to a hijack). Default value = None, which means no deletion.
Updates can be cleared periodically from the db using a clock signal (e.g., every hour)

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by slowr**
### Description of PR

Changed entrypoint of database's container to start the cron daemon by itself and also we map a new file that cleans all non-hijack BGP updates.

What component(s) does this PR affect?

- [x] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [x] Build System

Does the PR require changes on other components? If yes, please mark the components:

- [ ] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

### Related Issue

Resolves #33

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] None of the above

### Checklist:

- [x] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: other/postgres-cleanup.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+psql -d $POSTGRES_DB -U $POSTGRES_USER -c "DELETE FROM bgp_updates WHERE timestamp < NOW() - interval '1 hours' AND handled=true AND hijack_key=ARRAY[]::text[];"
```
Comment (vkotronis) [inline other/postgres-cleanup.sh]: please make this configurable with an env variable (preferrably in number of days). Also remove unhandled updates too, no need to keep them in

File: other/postgres-entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+if [[ "${DB_BACKUP}" == "true" ]]; then
+    cat > /etc/periodic/daily/backup <<EOF
+#!/bin/sh
+pg_dump -d $POSTGRES_DB -U $POSTGRES_USER -F t -f /tmp/db.tar > /tmp/db.log 2>&1
+EOF
+fi
+
+re='^[0-9]+$'
+if [[ $DB_AUTOCLEAN =~ $re ]]; then
```
Comment (vkotronis) [inline other/postgres-entrypoint.sh]: If someone does not want autoclean he simply puts a None value there?

Comment (vkotronis) [reply other/postgres-entrypoint.sh]: please also add a default value (e.g., None) in the current .env file

File: other/postgres-entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+if [[ "${DB_BACKUP}" == "true" ]]; then
```
Comment (vkotronis) [inline other/postgres-entrypoint.sh]: plelase add default true value in the current .env file

Comment (vkotronis) [reply other/postgres-entrypoint.sh]: also  document in the wiki both new variables

File: other/postgres-entrypoint.sh (hunk 2 above)
Comment (slowr) [reply other/postgres-entrypoint.sh]: default will be none? now its 24

File: other/postgres-entrypoint.sh (hunk 3 above)
Comment (slowr) [reply other/postgres-entrypoint.sh]: its already set to true

**Pull request merged**